specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,165 @@
+
+**[1]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page |
+| `1` | virtual_page |
+<!-- endsemconv -->
+
+## Ajax
+
+<!-- semconv browser.ajax -->
+The event name MUST be `ajax`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `page.url` | string | The URL of the page that made the Ajax call | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Ajax request type | `0` | Required |
+| [`http.method`](../../../trace/semantic_conventions/http.md) | string | HTTP request method. | `GET`; `POST`; `HEAD` | Required |
+| [`http.request_content_length`](../../../trace/semantic_conventions/http.md) | int | The size of the request payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size. | `3495` | Recommended |
+| [`http.response_content_length`](../../../trace/semantic_conventions/http.md) | int | The size of the response payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size. | `3495` | Recommended |
+| [`http.status_code`](../../../trace/semantic_conventions/http.md) | int | [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6). | `200` | Conditionally Required: If and only if one was received/sent. |
+| [`http.url`](../../../trace/semantic_conventions/http.md) | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. [1] | `https://www.foo.bar/search?q=OpenTelemetry#SemConv` | Required |
+
+**[1]:** `http.url` MUST NOT contain credentials passed via URL in form of `https://username:password@www.example.com/`. In such case the attribute's value should be `https://www.example.com/`.
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | fetch |
+| `1` | xhr |
+<!-- endsemconv -->
+
+## Exception
+
+<!-- semconv browser.exception -->
+The event name MUST be `exception`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `file_name` | string | Name of the file that generated the error | `foo.js` | Recommended |
```

Comment:
Field name has an extra _. what is the thinking here?  Again differs from what is captured in the word doc.

Reply:
The yaml to md conversion tool did not allow camel cased attribute names, hence used the underscore equivalent. Let's discuss which format we want to use.

Reply:
Added the exception prefix.